.github/workflows/release-cut.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Cut
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  prep:
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.title, 'RELEASE')
```

Comment:
Once the release prep PR goes through, this action will trigger and run the `ge_releaser tag` command; this will kick off a release-enabled Azure build and automatically publish to PyPI.